index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="./src/styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>puppyHeads or puppyButts</title>
+</head>
+<body>
+    <header>
+        <div id="header">
+                <h1>Puppy Heads or Puppy Butts?</h1>
+        </div>
+    </header>
+        <main>
+                <p id="wins-losses-wallet">
+                    <span id="wins">wins: 0</span>
+                    
+                    <span id="losses">losses: 0</span>
+
+                    <p id="wallet">You have $150 in your wallet</p>
+                 </p>
+            <p>
+                <img id="puppy-image" src="./assets/Heads.jpg" alt="picture of puppy head">
+            </p>
+            <p id="message">Pick Heads or Tails and place a bet!</p>
+            <span>
```

Comment:
Spans should only be used for nested content or for setting different styles to text inside other tags. I would stay away from nesting other elements within spans for the most part.